tests/jballerina-unit-test/src/test/resources/test-src/statements/ifelse/type-guard.bal

```diff
@@ -668,18 +668,12 @@ function testFiniteTypeInUnionAsComplexFiniteTypes_4() returns boolean {
 }
 
 function testFiniteTypeInUnionAsComplexFiniteTypes_5() returns boolean {
-    FooBarOneTwoBoolean f = 1;
```

Comment:
For

```ballerina
type FooBarOneTwoBoolean "foo"|"bar"|1|2.0|boolean;

function fn() {
    FooBarOneTwoBoolean _ = 1;
}
```
an error is logged

```
incompatible types: expected 'FooBarOneTwoBoolean', found 'float'
```

This is not correct. The type of `1` here is `int`, so it will be compatible.